pacai/util/probability.py

```diff
@@ -51,23 +55,25 @@ def sample(distribution, values = None):
         distribution = [i[1] for i in items]
         values = [i[0] for i in items]
 
-    if sum(distribution) != 1:
+    if len(distribution) == 0:
+        raise ValueError("Empty distribution")
```

Comment:
I'm not a fan of these error messages.
We want it to be more descriptive.
Remember, students will be the main ones getting these errors.
(I also prefer messages that are full sentences.)

So here, something like: "Distribution to sample from must be non-empty."